site/en/r2/tutorials/text/nmt_with_attention.ipynb

```diff
@@ -584,7 +584,7 @@
         "\u003cimg src=\"https://www.tensorflow.org/images/seq2seq/attention_equation_0.jpg\" alt=\"attention equation 0\" width=\"800\"\u003e\n",
         "\u003cimg src=\"https://www.tensorflow.org/images/seq2seq/attention_equation_1.jpg\" alt=\"attention equation 1\" width=\"800\"\u003e\n",
         "\n",
-        "We're using *Bahdanau attention*. Lets decide on notation before writing the simplified form:\n",
+        "Here, we're using *Bahdanau attention* in this encoder implement (paper: https://arxiv.org/pdf/1409.0473.pdf). Lets decide on notation before writing the simplified form:\n",
```

Comment:
This tutorial is using [Bahdanau attention](https://arxiv.org/pdf/1409.0473.pdf)